docs/source/building-supergraphs/subgraphs-apollo-server.mdx

```diff
@@ -206,42 +239,55 @@ The method for defining custom directives differs slightly for a federated graph
 > * Because directives are specific to individual subgraphs, it's valid for different subgraphs to define the _same_ directive with _different_ logic. Composition does not detect or warn about such inconsistencies.
 > * If multiple subgraphs can resolve a particular field, each subgraph should almost always apply the exact same set of custom directives (with the exact same accompanying logic) to that field. Otherwise, the behavior of that field might vary depending on _which_ subgraph resolves it.
 
-### Directives in Apollo Server 3.x
+### Directives in Apollo Server 4.x and 3.x
 
-Apollo Server 3 does not provide _built-in_ support for custom directives, but you can install certain `@graphql-tools` libraries to enable support. To get started with these libraries in Apollo Server, first read [Creating schema directives](https://www.apollographql.com/docs/apollo-server/schema/creating-directives/).
+Apollo Server versions 3 and 4 do not provide _built-in_ support for custom directives, but you can install certain `@graphql-tools` libraries to enable support. To get started with these libraries in Apollo Server, first read [Creating schema directives](https://www.apollographql.com/docs/apollo-server/v3/schema/creating-directives/).
```

Comment:
This link also works without the `v3` though more surprisingly to me than the previous one I mentioned since that doc no longer exists. So probably keeping the v3 here is correct, but noting my discovery in case that's interesting.

Reply:
Oh, bizarre! That shouldn't work. 🤔